sdks/go/pkg/beam/core/runtime/graphx/schema/schema_test.go

```diff
@@ -161,18 +354,22 @@ func TestSchemaConversion(t *testing.T) {
 				if err != nil {
 					t.Fatalf("error ToType(%v) = %v", test.st, err)
 				}
-
-				if d := cmp.Diff(reflect.New(test.rt).Elem().Interface(), reflect.New(got).Elem().Interface()); d != "" {
-					t.Errorf("diff (-want, +got): %v", d)
+				if !test.rt.AssignableTo(got) {
+					t.Errorf("%v not assignable to %v", test.rt, got)
+					if d := cmp.Diff(reflect.New(test.rt).Elem().Interface(), reflect.New(got).Elem().Interface()); d != "" {
+						t.Errorf("diff (-want, +got): %v", d)
+					}
 				}
 			}
 			{
 				got, err := FromType(test.rt)
 				if err != nil {
 					t.Fatalf("error FromType(%v) = %v", test.rt, err)
 				}
-
-				if d := cmp.Diff(test.st, got); d != "" {
+				if d := cmp.Diff(test.st, got,
+					protocmp.Transform(),
+					protocmp.IgnoreFields(proto.MessageV2(&pipepb.Schema{}), "id", "options"),
```

Comment:
I get why this is ignoring the "id" field for the proto, but why "options"? This test seems to already be testing options in the other direction (options are respected when converting from schemas to reflect types). I figure it would also want to test that options are included as expected when converting to schemas.

Reply:
I'll try removing it before submitting. 
I might have added this at an earlier iteration when the options also contained the fully qualified paths for comparison, which could vary from machine to machine. Those aren't being saved in the options anymore so it's fair for use to get rid of at this point I think....

Reply:
As a note, it's good practice to approve but still request small changes when the other person is able to merge themselves. As committers we're already responsible to ensure the code is good for the project, but allowing folks to be unblocked is good for the community.